qiskit/transpiler/target.py

```diff
@@ -502,6 +502,28 @@ def operation_names_for_qargs(self, qargs):
             raise KeyError(f"{qargs} not in target.")
         return self._qarg_gate_map[qargs]
 
+    def instruction_supported(self, operation_name, qargs):
+        """Return whether the instruction (operation + qubits) is supported by the target
+
+        Args:
+            operation_name (str): The name of the operation for the instruction
+            qargs (tuple): The tuple of qubit indices for the instruction
+
+        Returns:
+            bool: Returns ``True`` if the instruction is supported and ``False`` if it isn't.
+
+        """
+        # Case a list if passed in by mistake
```

Comment:
```suggestion
        # Cast a list if passed in by mistake
```

Maybe?

Reply:
Yeah it should be that, but this was part of #7778 so we can't fix it as part of this PR (it's probably only showing up in the incremental diff because of the merge/rebase in https://github.com/Qiskit/qiskit-terra/pull/7776/commits/b4afbc01bc24dc1196e9cbd8df3e628910a8bdc2).

Reply:
Ah yeah that's exactly what happened.